trio/_tests/test_subprocess.py

```diff
@@ -488,7 +515,7 @@ def test_waitid_eintr():
     got_alarm = False
     sleeper = subprocess.Popen(["sleep", "3600"])
 
-    def on_alarm(sig, frame):
+    def on_alarm(sig: object, frame: object) -> None:
```

Comment:
Isn't `sig` a signal type object and `frame` a `FrameType` (from `types`) object?

Reply:
:+1: